Formula/makensis.rb

```diff
@@ -32,8 +39,13 @@ def install
     # https://sourceforge.net/p/nsis/bugs/1085/
     ENV.libstdcxx if ENV.compiler == :clang
 
+    # requires zlib (win32) to build utils
+    resource("zlib-win32").stage do
```

Comment:
@dunn I tried this (think I tried this before), but it fails to build

```sh
==> Downloading https://downloads.sourceforge.net/project/libpng/zlib/1.2.8/zlib128-dll.zip
Already downloaded: /Users/brew/Library/Caches/Homebrew/makensis--zlib-win32-dll.zip
==> Verifying makensis--zlib-win32-dll.zip checksum
unzip -qq /Users/brew/Library/Caches/Homebrew/makensis--zlib-win32-dll.zip
Error: Permission denied - (/private/tmp/makensis--zlib-win32-20170510-38621-3cuyd5/include, /private/tmp/makensis-20170510-38621-1j8rz7f/nsis-3.01-src/zlib/include)
```
[full log](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1162/version=el_capitan/console)